maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRewritePomsPhase.java

```diff
@@ -89,6 +92,18 @@ public abstract class AbstractRewritePomsPhase extends AbstractReleasePhase impl
      */
     private String modelETL = JDomModelETLFactory.NAME;
 
+    /**
+     * Regular expression pattern matching Maven expressions (i.e. references to Maven properties).
+     * The first group selects the property name the expression refers to.
+     */
+    private static final Pattern EXPRESSION_PATTERN = Pattern.compile("\\$\\{(.+)\\}");
```

Comment:
Would fail in case of `<version>${revision}${sha1}${changelist}</version>`, looks like the `revision` the only property from CI Friendly is supported right now.

Reply:
https://github.com/apache/maven-release/pull/202